src/main/java/uk/ac/ebi/biosamples/jsonschemastore/service/FieldRepositoryEventHandler.java

```diff
@@ -45,6 +45,9 @@ public void handleBeforeSave(Field field) {
     String oldFieldId = field.getId();
     Field oldField = fieldRepository.findById(field.getId())
         .orElseThrow(() -> new DataIntegrityViolationException("Could not find the field: " + oldFieldId));
+    if(!oldField.isLatest()) {
+      throw new IllegalStateException("Non latest versions are not updatable");
```

Comment:
What http error code is returned to the user? 501?

Reply:
403 Forbidden.